src/java/org/apache/cassandra/index/sai/disk/io/IndexFileUtils.java

```diff
@@ -75,9 +79,20 @@ public IndexInput openBlockingInput(File file)
         return IndexInputReader.create(randomReader, fileHandle::close);
     }
 
+    public ChecksumIndexInput getBufferedChecksumIndexInput(IndexInput indexInput)
```

Comment:
nit: Just make this `static` so we don't need to refer to the `instance`?

Reply:
Fixed.